index.js

```diff
@@ -1,53 +1,47 @@
 // Load dotenv first
 require('dotenv').config({ silent: true });
 
-const utility = require('./src/utility');
 const Accuser = require('accuser');
 
-let currentLevel = require('./config')['currentLevel'];
+let currentLevel = require('./config').currentLevel;
 let semesterAccount = require('./config').semesterAccount;
 
-const accuser = new Accuser({  interval: 600000 });
+const accuser = new Accuser({ interval: 600000 });
 
 const githubAuthToken = {
-  "type": "oauth",
-  "token": process.env.GITHUB_TOKEN
+  type: 'oauth',
+  token: process.env.GITHUB_TOKEN
 };
 
 accuser.authenticate(githubAuthToken);
 
-// this section of code initializes Accuser for the current semester's repository
-// currentLevel in data.json determines which highest level repository is already
-//    made available to the students.
-// previous semesters are no longer handled by this bot.
-let initializeSemesterRepositories = require('./src/semester');
-for (var level = 1; level <= currentLevel; ++level) {
-  initializeSemesterRepositories(accuser, 'addressbook-level' + level);
+// Whitelisted
+const SubmissionRepos = require('./src/whitelist');
```

Comment:
hmm i think convention maybe to name the file the same as you name your class, so if your class is `WhiteList` the file will be `WhiteList.js`

Reply:
Btw better to move all the require to the top of the file